source/includes/usage-examples/code-snippets/findOne.go

```diff
@@ -35,16 +35,18 @@ func main() {
 
 	// begin findOne
 	coll := client.Database("sample_mflix").Collection("movies")
+	
```

Comment:
Adding a line here broke your emphasize-lines setting. I wasn't able to check many of these since it's not that apparent which code snippets correspond to which page in all cases. Could you check that they all render correctly as expected, and fix the ones that aren't quite right like this one?